cpp/include/cudf/utilities/pinned_memory.hpp

```diff
@@ -55,4 +55,20 @@ struct pinned_mr_options {
  */
 bool config_default_pinned_memory_resource(pinned_mr_options const& opts);
 
+/**
+ * @brief Set the threshold size for using kernels for pinned memory copies.
+ *
+ * @param threshold The threshold size in bytes. If the size of the copy is less than this
+ * threshold, the copy will be done using kernels. If the size is greater than or equal to this
+ * threshold, the copy will be done using cudaMemcpyAsync.
+ */
+void kernel_pinned_copy_threshold(size_t threshold);
```

Comment:
this is an undefined function, it should be `set_kernel_pinned_copy_threshold`.

Reply:
weird, not sure how that happened
fixed